lzy/allocator/src/main/java/ai/lzy/allocator/alloc/dao/DynamicMountDao.java

```diff
@@ -0,0 +1,29 @@
+package ai.lzy.allocator.alloc.dao;
+
+import ai.lzy.allocator.model.DynamicMount;
+import ai.lzy.model.db.TransactionHandle;
+import org.jetbrains.annotations.Nullable;
```

Comment:
we use `jakarta` annonations

Reply:
Why? Jetbrains annotations are tracked by IDEA and it helps to highlight possible NPE. But I don't see much benefit from jakarta/javax annotations

Reply:
jakarta annotations do the same (just configure IDE) :)
![2023-04-12_16-09](https://user-images.githubusercontent.com/9839826/231468693-6c1c2df4-6bc2-4c1e-be69-53039e3284ff.png)

Reply:
![2023-04-12_16-15](https://user-images.githubusercontent.com/9839826/231469186-b21bdc5f-5b53-4b0f-b810-734b79785f76.png)

Reply:
it seems that we will have to abandon the use of JetBrains IDE

Reply:
Returned jakarta annotations